cylc/uiserver/resolvers.py

```diff
@@ -57,6 +58,16 @@
 }
 
 
+LOG_FILE_MAPPINGS = {
+    'job': 'j',
+    'job.out': 'o',
+    'job.err': 'e',
+    'job-activity.log': 'a',
+    'job.status': 's',
+    'job.xtrace': 'x'
+}
+
+
```

Comment:
Can import this mapping from `cylc.flow.task_job_logs`, this will ensure the mapping is correct for the version of `cylc cat-log` being used.

Note it's back to front in cylc-flow but can be easily reversed:

```python
{
    value: key
    for key, value in x.items()
}
```